### ISSUE TITLE
Länken "Min förvaltning" död när man besöker Komin från utanför vårt eget nätverk

### ISSUE THREAD

**Issue by perber14** [malmostad/global_assets#8]
Länken saknar "https://" och leder ingenstans, utan visar "sidan kan inte visas".

**Comment by martent** [comment]
Är det när du klickar i på *Min fförvaltning* sidhuvudet? Varje roll (Förvaltning eller Arbetsfält) har en hemsida som man anger i administrationen för roller på Min sida. Administratörsroll på Min sida/Kontaktboken behövs.

Exempel på redigeringsvy:
https://webapps06.malmo.se/dashboard/roles/30/edit

**Comment by perber14** [comment]
Ja, precis. Menar du att det kan ha att göra med felaktiga inställningar? För det funkar som det ska när jag är inne i vårt nätverk, det är bara när jag är inne utifrån det tjallar.

**Comment by martent** [comment]
Skadar ju inte att ändra till https: i rolladministrationen eftersom det är det som är korrekt adress. Det kan vara nått i åtkomstlösningen som inte tillåter trafik utan ssl och att redirect-funktionen från http till https på servern inte ens nås nätmässigt.

Det var inte så att Komin-SV var nere när du klickade, har det hänt fler gånger?

Funkar inte ovanstående får det felsökas av de nya konsulterna. De kommer antagligen att behöva en "plattformsdator" för att upprepa felet, på rätt sätt.

**Comment by perber14** [comment]
1. Vem gör det`Idag Sven, på måndag Magnus?

2.  Nej, det var inte nere i allmänhet

**Comment by perber14** [comment]
Sven låter meddela att han tror att detta problem endast gäller plattformsdatorer som inte är anslutna till vårt eget nätverk.

**Comment by magnivar** [comment]
kolla länkarna så de är https

**Comment by martent** [comment]
Jag behöver tillgång till en plattformsdator för att kunna upprepa och diagnostisera felet. Det funkar på alla mina enheter och uppkopplingar.

**Comment by magnivar** [comment]
Jag kollar vad som gäller nu.
